src/user-guide/running-workflows/tasks-jobs-ui.rst

```diff
@@ -102,31 +102,32 @@ The "n" Window
 
 .. versionchanged:: 8.0.0
 
-Cylc workflow :term:`graphs <graph>` can be very large, even infinite for
-:term:`cycling workflows <cycling workflow>` with no :term:`final cycle point`.
+Cylc workflow :term:`graphs <graph>` can be very large, or infinite in
+extent for :term:`cycling workflows <cycling workflow>` with no
+:term:`final cycle point`.
 
 Consequently the GUI often can't display "all of the tasks" at once. Instead
-it displays all :term:`active tasks <active task>` (e.g. running tasks)
-as well as any tasks which are a configurable number of tasks away from
-them in the task dependency :term:`graph`.
+it displays all tasks in the :term:`active window` of the workflow, as well
+as any tasks out to a configurable number of graph edges away from them in
+the task dependency :term:`graph`.
 
 .. image:: ../../img/n-window.png
    :align: center
 
 
 n=0:
-   The ``n=0`` window contains :term:`active tasks <active task>`. An active
-   task is a task which is near ready to run, in the process of running, or
-   which requires user intervention (see the :term:`glossary <active task>`
-   for a more detailed description).
+   The ``n=0`` window corresponds to the scheduler's active window:
+   tasks that are near ready to run, in the process of running, or
```

Comment:
Pedantic, this needs to be a complete sentence, or you need semicolon on the line about.

While my suggestion risks comic over-repitition of the words `active window` I think it makes the meaning clearer.

```suggestion
   The active window contains tasks that are near ready to run, in the process of running, or
```

Reply:
> Pedantic, this needs to be a complete sentence, or you need semicolon on the line about.

I don't think that's right. The form of my sentence is *"The x corresponds to y: <description of y"*.